src/DevBetterWeb.Core/Entities/MemberVideo.cs

```diff
@@ -0,0 +1,30 @@
+﻿using DevBetterWeb.Core.Interfaces;
+using DevBetterWeb.Core.SharedKernel;
+
+namespace DevBetterWeb.Core.Entities
+{
+  public class MemberVideo : BaseEntity, IAggregateRoot
```

Comment:
Maybe renamed to `MemberVideoProgress` ?

Reply:
Yes, that is better.
Done